src/routes/admin-api/route.ts

```
@@ -625,6 +626,164 @@ adminApiRoutes.get("/models", (c) => {
+function parseNonEmptyString(value: unknown): string | null {
+  if (typeof value !== "string") return null
+  const trimmed = value.trim()
+  return trimmed.length > 0 ? trimmed : null
+}
+
+function parseOptionalFiniteNumber(value: unknown): number | undefined {
+  if (typeof value !== "number") return undefined
+  return Number.isFinite(value) ? value : undefined
+}
+
+function parseOptionalBoolean(value: unknown): boolean | undefined {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'parseOptionalBoolean'. Consider to delete it or rename it.

'parseOptionalBoolean' is defined here:


(lint/suspicious/noRedeclare)